apps/engine/src/buildOptions.ts

```diff
@@ -26,7 +26,7 @@ export const buildUseCacheOption = <T extends {}>(y: Argv<T>) =>
 	});
 
 // eslint-disable-next-line @typescript-eslint/ban-types
-export const buildOptions = <T extends {}>(y: Argv<T>) => {
+export const buildOptions = <T extends Record<string, unknown>>(y: Argv<T>) => {
```

Comment:
Why is this change required ?

Reply:
it's not really required, it's just to get rid of eslint comments